### ISSUE TITLE
Note that in the scotgov_management dataset, I've converted all *'s (which denote <5 ?) to zero!!

### ISSUE THREAD

**Issue by soniamitchell** [ScottishCovidResponse/SCRCdata#23]

**Comment by IainM-BioSS** [comment]
Thanks for the flag on  this, Sonia.  Are we confident that all recorded * are indeed <5 events?  How do they record missing data? Do they denote observations in the range 0<=x<5 or 0<x<5 .. ie are there also 'genuine' zeros?   Some of the inference models could, in principle cope with censored data.  What matters is that we distibguish, if possible, between zeros, <5 and missing values.

**Comment by soniamitchell** [comment]
I am pretty confident – but this will need to be checked. I could reinstate the * to distinguish between those values, 0s and NAs?

**Comment by IainM-BioSS** [comment]
No, it's fine for now, especially since you have flagged it as an important issue that users need to be aware of.  But it is exactly the type of situation where we just need to be a wee bit careful.  There is often an issue distinguishing between zeros and missign values, and the censored element just makes it more fiddly.  If we are confident that there are *no* zeros.. ie that they are all included in the * terms, then coding them all as zero is very much the right thing to have done.  But sense checking the data for zeros and missing values would be a good idea.

**Comment by soniamitchell** [comment]
I'll add an issue to the database
